src/main/scala/edu/cmu/cs/obsidian/codegen/yulAST.scala

```diff
@@ -232,7 +243,10 @@ case class YulObject(name: String, code: Code, subobjects: Seq[YulObject], data:
                         dispatchArray = dispatchArray :+ codegen.Case(hexlit(hashOfFunctionDef(f)), Block(dispatchEntry(f)))
                     case e: ExpressionStatement =>
                         e.expression match {
-                            case f: FunctionCall => memoryInitRuntime = f.toString
+                            case f: FunctionCall =>
+                                //TODO what was this line doing?
+                                //memoryInitRuntime = f.toString
```

Comment:
If you don't know, why remove it?

Reply:
I think it's dead code, but I'm not positive.